wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1633,6 +1633,40 @@ class SwiftGenerator private constructor(
 
     fun builtInType(protoType: ProtoType): Boolean = protoType in BUILT_IN_TYPES.keys
 
+    private val SWIFT_COMMON_TYPES = setOf(
```

Comment:
Not something we want swift poet to do?

Reply:
That's what the name allocator is for

Reply:
In the PR description I wrote:

> There is a chance this can be fixed in SwiftPoet by building in this mechanism into it instead of Wire but for now this is a cheap lookup Wire can do.

If NameAllocator is the correct path to go then yes SwiftGenerator still does not use it so we could delete this in the near future when NameAllocator is used.

Reply:
I believe once this TODO is resolved https://github.com/outfoxx/swiftpoet/blob/main/src/main/java/io/outfoxx/swiftpoet/NameAllocator.kt#L121-L123 we can adopt the SwiftPoet NameAllocator here.